### PR TITLE
[Serializer] ObjectNormalizer, object_to_populate doc. Minor enhancements.

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes |
| Applies to | 2.7 |
| Fixed tickets | partially #4698 |

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -425,6 +455,9 @@ If you are using isser methods (methods prefixed by ``is``, like
 ``Acme\Person::isSportsman()``), the Serializer component will automatically
 detect and use it to serialize related attributes.
 
+The ``ObjectNormalizer`` also take care of methods starting by ``has``, ``add``
```
Comment (xabbuh) [inline components/serializer.rst]: takes care

Comment (xabbuh) [inline components/serializer.rst]: starting with

File: components/serializer.rst [hunk 2]
```diff
@@ -461,23 +494,42 @@ Normalizers
...
-    Objects are serialized to a map of property names (method name stripped of
+    Objects are normalized to a map of property names (method name stripped of
     the "get" prefix and converted to lower case) to property values.
 
 :class:`Symfony\\Component\\Serializer\\Normalizer\\PropertyNormalizer`
     This normalizer directly reads and writes public properties as well as
-    **private and protected** properties. Objects are normalized to a map of
-    property names to property values.
+    **private and protected** properties.
+    It supports calling the constructor during the denormalization process.
```
Comment (xabbuh) [inline components/serializer.rst]: You need to add a blank line to get a new paragraph.

File: components/serializer.rst [hunk 3]
```diff
@@ -461,23 +494,42 @@ Normalizers
...
+
+    Objects are normalized to a map of property names to property values.
 
 .. versionadded:: 2.6 The
     :class:`Symfony\\Component\\Serializer\\Normalizer\\PropertyNormalizer`
     class was introduced in Symfony 2.6.
 
+.. versionadded:: 2.7 The
```
Comment (xabbuh) [inline components/serializer.rst]: "The" should be one the next line.

**Comment by dunglas** [comment]
@xabbuh commend addressed.

File: components/serializer.rst [hunk 4]
```diff
@@ -43,18 +46,18 @@ which Encoders and Normalizer are going to be available::
...
     $encoders = array(new XmlEncoder(), new JsonEncoder());
-    $normalizers = array(new GetSetMethodNormalizer());
+    $normalizers = array(new ObjectNormalizer());
 
     $serializer = new Serializer($normalizers, $encoders);
 
-There are several normalizers available, e.g. the
-:class:`Symfony\\Component\\Serializer\\Normalizer\\GetSetMethodNormalizer` or
-the :class:`Symfony\\Component\\Serializer\\Normalizer\\PropertyNormalizer`.
+The preferred normalizer is
```
Comment (xabbuh) [inline components/serializer.rst]: [...] is the [...]

File: components/serializer.rst [hunk 5]
```diff
@@ -43,18 +46,18 @@ which Encoders and Normalizer are going to be available::
...
+:class:`Symfony\\Component\\Serializer\\Normalizer\\ObjectNormalizer` but other
```
Comment (xabbuh) [inline components/serializer.rst]: [...] ObjectNormalizer, but [...]

**Comment by dunglas** [comment]
@xabbuh done

**Comment by weaverryan** [comment]
Really really nice - a lot of things updated here. Awesome!

What's left on #4698? What other plans do you have or areas you think are lacking?

**Pull request merged**